### PR TITLE
Adds modified version to scap S3 path for non-tag deployments

### PR DESCRIPTION
**Pull request by eemperor**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -114,7 +115,7 @@ jobs:
             tags: true
         - provider: script
           # publish dev scap reports
-          script: bash ci/codebuild_start_wait.sh $CODEBUILD_PROJECT s3://$DEV_BUCKET/$S3_KEYFIX/$RELEASE_VERSION/scans
+          script: bash ci/codebuild_start_wait.sh $CODEBUILD_PROJECT s3://$DEV_BUCKET/$S3_KEYFIX/$CURRENT_VERSION/scans
```
Comment (lorengordon) [inline .travis.yml:118]: For both cases, here and RELEASE_VERSION, I think instead we can just use the env provided by gravitybee...

```suggestion
          script: bash ci/codebuild_start_wait.sh $CODEBUILD_PROJECT s3://$DEV_BUCKET/$S3_KEYFIX/$GB_ENV_APP_VERSION/scans
```

**Review by lorengordon — CHANGES_REQUESTED** [review]
Gravitybee gives us an env we can use instead...

File: .travis.yml (hunk 1 above)
Comment (eemperor) [reply .travis.yml]: Good call.  Switched both to `GB_ENV_APP_VERSION`.

**Review by lorengordon — APPROVED** [review]
🚢

**Pull request merged**